### DISCUSSION TITLE
雷电模拟器地址似乎是这个

### DISCUSSION THREAD

**Discussion by SekiBetu** [MaaAssistantArknights/MaaAssistantArknights#603] [Show and tell]
![image](https://user-images.githubusercontent.com/38283893/170849334-26aef94b-b7cc-4d90-9098-ab22154de764.png)
![image](https://user-images.githubusercontent.com/38283893/170849339-071ee8d5-8bf3-4363-a1c6-85bc309c70ab.png)
如果填127.0.0.1:5555会连接失败
希望可以在主页readme告诉一下大家先用adb devices去测一下地址再填

**Comment by MistEO** [comment]
这俩会来回跳，我没找到规律。有些人电脑上雷电就是 `127.0.0.1:5555`，甚至会有这次重启是 `127.0.0.1:5555`, 但下次重启又变回 `emulator-5554` 的情况（

**Reply by MistEO** [reply]
https://github.com/MaaAssistantArknights/MaaAssistantArknights/issues/686
